schema.sql

```diff
@@ -101,3 +101,16 @@ CREATE TABLE visits(
   vet_id INTEGER REFERENCES vets(id),
   date_of_visits DATE
 );
+
+-- Add an email column to your owners table
+
+ALTER TABLE owners ADD COLUMN email VARCHAR(120);
+
+-- Improve second EXPLAIN ANALIZE query
+
+CREATE INDEX idx_animal_id ON visits(animal_id);
+
+
```

Comment:
- [ ] Good job adding the create index queries but please kindly include that for email on the owners table. This will help improve the execution time.

Reply:
Hello @AnselemOdims, thank you for the review. Apparently, we forgot to commit the improvement of the email column, but now we have solved the problem 😁